Lastline v2 - Fixed the detailed description (#7844)
demisto/content: Packs/Lastline/Integrations/Lastline_v2/Lastline_v2_description.md

Provides threat analysts and incident response teams with the advanced
malware isolation and inspection environment, needed to safely execute advanced
malware samples and understand their behavior. 
API key and API token are generate by Lastline and given at the subscription time.

### How to use the `threshold` parameter.

> by default `threshold` is set to 70. it means that for every file with severity over 70
> the Dbot score will be high. you can change it as you want.

Reviewer: Let's fix this as well.

```suggestion
By default, the `threshold` parameter is set to 70. This means that for every file with a severity of over 70
the Dbot score will be set to *high*. This parameter is configurable.
```
